WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -64,4 +66,25 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleScrollCommand:(FBRouteRequest *)request
+{
+    NSString *const direction = request.arguments[@"direction"];
+    if (direction) {
+        if ([direction isEqualToString:@"up"]) {
```

Comment:
Maybe we could extract some of this logic to separate method and reuse it here and in element scroll command?